e2e/cypress/support/ui_commands.js

```diff
@@ -18,6 +18,11 @@ Cypress.Commands.add('toMainChannelView', (username, password) => {
     cy.get('#post_textbox').should('be.visible');
 });
 
+Cypress.Commands.add('toTopPublicChannelView', () => {
+    cy.visit('/');
+    cy.get('#publicChannelList > li').eq('2').click();
```

Comment:
`.eq(2)` with get the third `li` under `#publicChannelList`. Did you mean `eq(1)`?

Alternatively, this selector and code should more directly get just those that are with the class of sidebar item:

`cy.get('#publicChannelList .sidebar-item').first().click()`

Reply:
The issue with this is that the `publicChannelList` element's first child is the `PUBLIC CHANNELS    +` row, which then means we have to retrieve the second child of that (which would mean that both your suggestions wouldn't really work)

Would there be any recommended ways to retrieve the second child of an element aside from what I used over there?

Reply:
Actually, given your other feedback with regards to adding a new post to make sure there will always be a post in the channel, I figured it would be unnecessary to keep this method, as its purpose was to navigate to a channel that already had posts in it.

Should be a cleaner way to resolve this